contentcuration/contentcuration/static/js/edit_channel/file_upload/views.js

```diff
@@ -500,7 +500,10 @@ var FormatEditorItem = BaseViews.BaseListNodeItemView.extend({
   },
   create_thumbnail_view: function(onstart, onfinish, onerror) {
     if (!this.thumbnail_view) {
-      var preset_id = _.findWhere(this.model.get('associated_presets'), { thumbnail: true }).id;
+      var preset_id = _.findWhere(Constants.FormatPresets, {
```

Comment:
Use a single source of truth from the constants file rather than the loadconstants output